cifar10/vgg19.jl

```diff
@@ -0,0 +1,85 @@
+using MLDatasets
+using Flux
+using Flux: onehotbatch, argmax, crossentropy, throttle
+using Base.Iterators: repeated, partition
+
+# Fetching the train and test data and getting them into proper shape
+
+train_X, train_Y = CIFAR10.traindata()
+train_x = convert(Array{Float64,4},float.(train_X))
+labels = onehotbatch(train_Y, 0:9)
+
+test_X, test_Y = CIFAR10.testdata()
+test_x = convert(Array{Float64,4},float.(test_X))
+labels_test = onehotbatch(test_Y, 0:9)
+
+train = [(train_x[:,:,:,i], labels[:,i])
+         for i in partition(1:50000, 10)]
+
+# Defining the blocks of convolution layers
+
+conv1_1 = Conv((3,3), 3=>64, relu, pad=(1,1), stride=(1,1))
+conv1_2 = Conv((3,3), 64=>64, relu, pad=(1,1), stride=(1,1))
+
+conv2_1 = Conv((3,3), 64=>128, relu, pad=(1,1), stride=(1,1))
+conv2_2 = Conv((3,3), 128=>128, relu, pad=(1,1), stride=(1,1))
+
+conv3_1 = Conv((3,3), 128=>256, relu, pad=(1,1), stride=(1,1))
+conv3_2 = Conv((3,3), 256=>256, relu, pad=(1,1), stride=(1,1))
+conv3_3 = Conv((3,3), 256=>256, relu, pad=(1,1), stride=(1,1))
+conv3_4 = Conv((3,3), 256=>256, relu, pad=(1,1), stride=(1,1))
+
+conv4_1 = Conv((3,3), 256=>512, relu, pad=(1,1), stride=(1,1))
+conv4_2 = Conv((3,3), 512=>512, relu, pad=(1,1), stride=(1,1))
+conv4_3 = Conv((3,3), 512=>512, relu, pad=(1,1), stride=(1,1))
+conv4_4 = Conv((3,3), 512=>512, relu, pad=(1,1), stride=(1,1))
+
+conv5_1 = Conv((3,3), 512=>512, relu, pad=(1,1), stride=(1,1))
+conv5_2 = Conv((3,3), 512=>512, relu, pad=(1,1), stride=(1,1))
+conv5_3 = Conv((3,3), 512=>512, relu, pad=(1,1), stride=(1,1))
+conv5_4 = Conv((3,3), 512=>512, relu, pad=(1,1), stride=(1,1))
+
+# Building the model
+
+m = Chain(
+	conv1_1,
+    conv1_2,
+    x -> maxpool(x, (2,2)),
```

Comment:
what about 
```julia
Chain( 
    Conv((3,3), 3=>64, relu, pad=(1,1), stride=(1,1)),
    Conv((3,3), 64=>64, relu, pad=(1,1), stride=(1,1)),
.....
```
to save some space?

Reply:
Made the changes that have been suggested